flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/DefaultExecutionGraph.java

```diff
@@ -59,8 +59,10 @@
 import org.apache.flink.runtime.jobgraph.JobVertex;
 import org.apache.flink.runtime.jobgraph.JobVertexID;
 import org.apache.flink.runtime.jobgraph.tasks.CheckpointCoordinatorConfiguration;
+import org.apache.flink.runtime.jobmanager.scheduler.SlotSharingGroup;
```

Comment:
Looks to me commit 6cb03fe1f9824c011f752a416197d1e2004a2e11 (Support calcuate network memory for dynamic graph.) is not related to the topic of this PR "Let some scheduler components updatable"?
If not, would you create a separate ticket and PR to make this change?

Reply:
I have separated the "Support calculate network memory for dynamic graph" and "Support register operator coordinators for new initialized ExecutionJobVertex" into two new tickets: FLINK-25668 and FLINK-25669